homeassistant/components/tuya/select.py

```diff
@@ -342,6 +342,31 @@
             translation_key="countdown",
         ),
     ),
+    # Dehumidifier
+    # https://developer.tuya.com/en/docs/iot/categorycs?id=Kaiuz1vcz4dha
+    "cs": (
+        SelectEntityDescription(
+            key=DPCode.COUNTDOWN_SET,
+            name="Countdown",
+            entity_category=EntityCategory.CONFIG,
+            icon="mdi:timer-cog-outline",
+            translation_key="countdown",
```

Comment:
What does this countdown entail?
Does it change value / actually count down when active?

Reply:
It's says how long the dehumidifier should run after powered on. 
If I set it to 3h, it doesn't show 2h after an hour, the value doesn't change.